billing-api/billing/__init__.py

```diff
@@ -82,6 +83,7 @@ def get_projects(client, user_id):
 @authenticate
 def generate_report_data(client, user_id):
     projects = request.args.get('projects')
+    user = request.args.get('user')
```

Comment:
If you do a `GET` here with no query params, you will a 500 error because python will throw a `TypeError`.  You need to do some input validation. 

<img width="709" alt="screen shot 2016-10-18 at 10 33 22 am" src="https://cloud.githubusercontent.com/assets/6350043/19482187/501da4b6-951e-11e6-89ff-bb7999769bff.png">